tests/migrations/test_commands.py

```diff
@@ -1732,6 +1732,44 @@ def test_squashmigrations_squashes(self):
             '  you can delete them.\n' % squashed_migration_file
         )
 
+
+    def test_squashmigrations_squashes_already_squashed(self):
+        out = io.StringIO()
+
+        with self.temporary_migration_module(
+            module="migrations.test_migrations_squashed_complex"
+        ) as migrations_dir:
+            call_command(
+                "squashmigrations",
+                "migrations",
+                "3_squashed_5",
+                "--squashed-name",
+                "double_squash",
+                interactive=False,
+            )
+
+            squashed_migration_file = os.path.join(
+                migrations_dir, "0001_double_squash.py"
+            )
+            # confirm the replaces mechanism holds the squashed migration (and not what it squashes, as the squash operations are what end up being used)
+            with open(squashed_migration_file, "r", encoding="utf-8") as fp:
```

Comment:
It's not evident everywhere in Django, but current code style is single quotes where possible.

Reply:
Moot point now since Django started using `black`, so disregard.